test/ts/migrations/3_deploy_test_contracts_test.ts

```diff
@@ -80,27 +87,37 @@ contract("Migration #3: Deploying Test Contracts", async (ACCOUNTS) => {
             expect(web3Utils.doesContractExistAtAddressAsync(mockTokenTransferProxy.address)).to
                 .eventually.be.true;
         });
+
         it("should deploy the `TokenRegistry` contract to the current network", async () => {
             tokenRegistry = await TokenRegistryContract.deployed(web3, TX_DEFAULTS);
             expect(web3Utils.doesContractExistAtAddressAsync(tokenRegistry.address)).to.eventually
                 .be.true;
         });
     });
+
     describe("#TokenRegistry", () => {
         it("should register the address for Augur's REP token", async () => {
             expect(
                 tokenRegistry.getTokenAddressBySymbol.callAsync(REP_TOKEN_SYMBOL),
             ).to.eventually.not.equal(NULL_ADDRESS);
         });
+
         it("should register the address for Maker's MKR token", async () => {
             expect(
                 tokenRegistry.getTokenAddressBySymbol.callAsync(MKR_TOKEN_SYMBOL),
             ).to.eventually.not.equal(NULL_ADDRESS);
         });
+
         it("should register the address for 0x's ZRX token", async () => {
             expect(
                 tokenRegistry.getTokenAddressBySymbol.callAsync(ZRX_TOKEN_SYMBOL),
             ).to.eventually.not.equal(NULL_ADDRESS);
         });
+
+        it("should register the address for the wrapped ETH token", async () => {
+            expect(
+                tokenRegistry.getTokenAddressBySymbol.callAsync(WRAPPED_ETH_SYMBOL),
+            ).to.eventually.not.equal(NULL_ADDRESS);
```

Comment:
I feel like these tests should be more general -- i.e. they should iterate over the list of tokens we have hardcoded in charta and make sure they're all registered.

Reply:
Perhaps separate PR / task though -- can you make a low priority task for this?